linera-views/src/test_utils/mod.rs

```diff
@@ -418,6 +418,38 @@ pub async fn run_writes_from_blank<C: LocalRestrictedKeyValueStore>(key_value_st
     }
 }
 
+/// Doing a big read of many keys could trigger some error. That need to be tested.
```

Comment:
"Reading many keys at a time could trigger an error. This needs to be tested." :)